src/main/py/hl7_listener/main.py

```diff
@@ -106,27 +150,39 @@ async def hl7_receiver():
         ) as hl7_server:
             # Listen forever or until a cancel occurs.
             await hl7_server.serve_forever()
+
     except asyncio.CancelledError:
         # Cancel errors are expected.
-        logger.info(logging_codes.HL7_MLLP_RECEIVER_CANCELLED)
+        logger.info(
+            HL7_MLLP_RECEIVER_CANCELLED,
+            logging_code="HL7LLOG005"
+        )
         pass
+
     except Exception as exp:
-        logger.error(logging_codes.HL7_MLLP_RECEIVER_ERR, exc_info=Exception(exception_formatter(str(exp))))
+        logger.error(
+            HL7_MLLP_RECEIVER_ERR,
+            logging_code="HL7LERR005",
+            exception=Exception(exception_formatter(str(exp)))
```

Comment:
I forgot that this service does this to keep phi out of logs..

For RAP team, typically the `exception` key & value are added by the lib, by default. Just a point of clarification you might want to make with them.